internal/view/cronjob.go

```diff
@@ -64,9 +67,106 @@ func jobCtx(path, uid string) ContextFunc {
 func (c *CronJob) bindKeys(aa ui.KeyActions) {
 	aa.Add(ui.KeyActions{
 		tcell.KeyCtrlT: ui.NewKeyAction("Trigger", c.trigger, true),
+		ui.KeyShiftS:   ui.NewKeyAction("Suspend", c.setSuspend, true),
```

Comment:
I think we should have a single action here to just toggle the job state. ie toggleSuspend. You can read the current state and just invert it in the handler. Would that make sense?

Reply:
Looks good!